portfolio/src/main/webapp/index.html

```diff
@@ -1,299 +1,293 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8" />
-    <title>Dalis Chan</title>
-    <link rel="stylesheet" href="assets/css/style.css" />
-    <script type="text/javascript" src="assets/js/script.js"></script>
-    <script type="text/javascript" src="assets/js/nav.js"></script>
 
-    <!-- FontAwesome icons -->
-    <script
-      src="https://kit.fontawesome.com/bf05d78857.js"
-      crossorigin="anonymous"
-    ></script>
-  </head>
-  <body>
-    <!-- Header -->
-    <header>
-        <div class="img-hover-zoom--colorize">
-          <img class="profile-img" src="images/profile-pic.jpg" alt="Avatar" />
+<head>
+	<meta charset="UTF-8" />
+	<title>Dalis Chan</title>
+	<link rel="stylesheet" href="assets/css/style.css" />
+	<script type="text/javascript" src="assets/js/script.js"></script>
+	<script type="text/javascript" src="assets/js/nav.js"></script>
+
+	<!-- FontAwesome icons -->
+	<script src="https://kit.fontawesome.com/bf05d78857.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+	<!-- Header -->
+	<header>
+		<div class="img-hover-zoom--colorize">
+			<img class="profile-img" src="images/profile-pic.jpg" alt="Avatar" />
         </div>
-        <h1>Dalis Chan</h1>
-        <p id="content">
-          Personal Portfolio
-        </p>
+			<h1>Dalis Chan</h1>
+			<p id="content">
+				Personal Portfolio
+			</p>
 
-    </header>
-    <!-- Navbar -->
-    <nav>
-      <ul>
-        <li><a href="#about">About Me</a></li>
-        <li><a href="#projects">Projects</a></li>
-        <li><a href="#work">Work Experiences</a></li>
-        <li><a href="#interests">Interests</a></li>
-      </ul>
-    </nav>
-    <main>
-      <section id="about">
-        <h1><i class="fas fa-laugh-beam"></i> About Me</h1>
-        <p id="content">
-          Hey there! I'm Dalis. I'm a Year 3 computer science undergrad at the
-          National University of Singapore. I love programming, especially web
-          development. I am currently interning as a Software Engineer at
-          Pawjourr x TWA for the 2020 summer. I love picking up new programming
-          skills and through this internship, two new things I am learning are
-          ReactJS and GraphQL. Read on more to find out what I do at school and
-          in my spare time!
-        </p>
-      </section>
-      <hr class="solid" />
-      <section id="projects">
-        <h1><i class="fas fa-code-branch"></i> My Projects</h1>
-        <p>
-          <span class="">I</span> love getting my hands dirty and making
-          software. I'm always excited to join hackathons or do interesting
-          projects!
-        </p>
+	</header>
+	<!-- Navbar -->
+	<nav>
+		<ul>
+			<li><a href="#about">About Me</a></li>
+			<li><a href="#projects">Projects</a></li>
+			<li><a href="#work">Work Experiences</a></li>
+			<li><a href="#interests">Interests</a></li>
+		</ul>
+	</nav>
+	<main>
+		<section id="about">
```

Comment:
The amount of spaces per indent seems quite high, are you able to adjust it? 2 or 4 spaces per indent should be plenty, and will help it fit in the text editor better :)